### ISSUE TITLE
Compiling multiple files fails if the path to that files contains space character

### ISSUE THREAD

**Issue by aleksaa01** [kriscross07/atom-gpp-compiler#221]
I posted my comment here: [https://github.com/kriscross07/atom-gpp-compiler/issues/7](url)

But then I thought I should post it as an issue, because I think that's not the intended behavior.

So this is the issue:

Issue for me was that the path to those files contained a space character, and because of that it couldn't compile.
Error:
'E:\C++_Projects\C++' is not recognized as an internal or external command, operable program or batch file.

And the path to those files was:
"E:\C++_Projects\C++ Example\example.cpp"
and
"E:\C++_Projects\C++ Example\example2.cpp"

Renaming "C++ Example" folder to "C++_Example", solved the problem...

**Comment by Zefty** [comment]
I am also having this issue, is there a way for ourselves to resolve this?

**Comment by aleksaa01** [comment]
> I am also having this issue, is there a way for ourselves to resolve this?

Yes, as I said up there, just remove spaces from your folders names.

But if I remember right, there are 2 functions for getting file paths: **getCompiledPath** and **getFilePath**.
And it's probably failing in **compileFile** function which is using both of these functions.

**Comment by lohart13** [comment]
It looks like line 403 of index.js is missing at least one double quote.
By changing this line to be ````const command = `start "${info.name}" cmd /C "${gdb ? "gdb" : ""} "${file}" ${gdb ? "" : "& echo. & pause"}"`;```` you then should be able to compile and run code from directories and files containing spaces.
